### PR TITLE
Add show/hide notification option

### LINKED ISSUE #1: Remove/disable notification that appears when new window is opened
Hi, a quick request for you: can you remove the notification that is shown when the context menu item is used to open a folder in a new window? (The one which says `Opening ${uri.path} in new window!`, that is).

It seems unnecessary since it is obvious that the new window has opened. Also, it's a bit bothersome, because it means I have to go back to the other window to dismiss it. I think it would be cleaner without it :)

### PR DESCRIPTION
**Pull request by notiv-nt**
Closes #1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +43,16 @@
...
 				}
 			]
+		},
+		"configuration": {
+			"title": "Open in New Window configuration",
+			"properties": {
+				"open-in-new-window.showNotification": {
+					"type": "boolean",
+					"description": "Show notification",
+					"default": false
```
Comment (notiv-nt) [inline package.json:53]: > It seems unnecessary

https://github.com/onatm/open-in-new-window/issues/1#issue-735415012

Therefore `false` as a default value

Comment (onatm) [reply package.json:53]: It is a sensible default value.

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (onatm) [inline .prettierrc]: Could you please remove this file?

**Review by onatm — CHANGES_REQUESTED** [review]
Thank you for your contribution @notiv-nt! It's looking great and I am ready to merge when you address the comment I made.

**Review by onatm — APPROVED** [review]

**Comment by onatm** [comment]
I'll release this asap in the morning. Thank you for your contribution @notiv-nt 🎉

**Comment by notiv-nt** [comment]
@onatm 

image

image

**Comment by onatm** [comment]
It's now published @notiv-nt 🎉

**Pull request merged**